test/index-test.js

```diff
@@ -110,6 +110,14 @@ tests = [{
     m: 'i'
   , o: [{i: 1, o: 2}, {i: 2, o: 2}]
   , e: [{i: 1}, {i: 2}]
+}, {
+    m: 'foo(bar)'
+  , o: { foo: { biz: 'baz' } }
+  , e: { foo: {} },
+}, {
+    m: 'foo(bar)'
+  , o: { foo: 1234 }
+  , e: { foo: {} }
```

Comment:
I don't think the object is the right response here because it changes the type of the input object (which is a number), and this library promises to not do that. I think a better response would be `null`, just as in the initial test cases.